probe-rs-tools/src/bin/probe-rs/cmd/dap_server/server/configuration.rs

```diff
@@ -141,6 +141,7 @@ impl SessionConfig {
             chip: self.chip.clone(),
             chip_description_path: self.chip_description_path.clone(),
             protocol: self.wire_protocol,
+            non_interactive: false,
```

Comment:
```suggestion
            non_interactive: true,
```

I'm not sure if you have a specific reason for `false` but I'm thinking that maybe we should treat the VS Code plugin as a non-interactive environment for now. I'm not sure how difficult it would be to show a dialog to select a probe, but either way that's out of the scope of this PR.

Reply:
Done